src/components/nav-bar/NavBar.tsx

```diff
@@ -0,0 +1,55 @@
+import classes from './NavBar.module.scss';
+
+import { GearSix, HouseSimple, Plus } from '@phosphor-icons/react';
+import { NavLink, useLocation } from 'react-router-dom';
+import { Paths } from '../../main';
+
+enum IconWeight {
+	ACTIVE = 'fill',
+	NOT_ACTIVE = 'light',
+}
+
+const BUTTON_ICON_WEIGHT = 'light';
+const ICON_SIZE = 32;
+
+const NavBar = () => {
+	const location = useLocation();
+
+	const getIconWeight = (isActive: boolean): IconWeight => {
+		return isActive ? IconWeight.ACTIVE : IconWeight.NOT_ACTIVE;
+	};
+
+	return (
+		<nav className={classes.navBar}>
+			<NavLink
+				to={Paths.HOME}
+				className={({ isActive }) => (isActive ? classes.active : '')}
+			>
+				<HouseSimple
+					weight={getIconWeight(location.pathname === Paths.HOME)}
+					size={ICON_SIZE}
+				/>
+				Home
+			</NavLink>
+			<NavLink to={Paths.ACTIVITIES}>
+				<Plus
+					weight={BUTTON_ICON_WEIGHT}
+					size={ICON_SIZE}
```

Comment:
I'm pretty sure it's possible to pass a string here that calls the `var()` css function instead of the ICON_SIZE constant, try that and see if it works, as it's better to have all size and color variables in the SCSS hierarchy instead.
If it doesn't let me know in a comment